helper-cli/src/main/kotlin/commands/GetPackageLicensesCommand.kt

```diff
@@ -121,7 +122,7 @@ class GetPackageLicensesCommand : CliktCommand(
 }
 
 private fun Collection<LicenseFinding>.toSpdxExpression(): String =
-    asSequence().map { it.license.toString() }.distinct().sorted().joinToString(" AND ")
+    asSequence().map { it.license.toString() }.distinct().joinToString(" AND ").toSpdx().sort().toString()
```

Comment:
There is no need to convert to string before concatenating and parsing back into an SPDX expression (could also be applied in the previous commit already):
```
asSequence().map { it.license }.reduce(SpdxExpression::and).sort().toString()
```
(not sure if `asSequence()` still makes sense with this code)

Reply:
> (not sure if `asSequence()` still makes sense with this code)

The IDE still suggested it, so I kept it.